test-util/src/main/java/io/camunda/zeebe/test/util/bpmn/random/blocks/ProcessBuilder.java

```diff
@@ -120,7 +122,23 @@ private void buildEventSubProcess(
   }
 
   public ExecutionPath findRandomExecutionPath(final Random random) {
-    final var followingPath = blockBuilder.findRandomExecutionPath(random);
+    final var startAnywhere = random.nextBoolean();
```

Comment:
:wrench: Not sure 50% chance is the right value here

Reply:
There is 2 different ways of starting a process so testing them both equally made sense to me. I did wonder whether we should increase the process count in the test now that we have more scenarios 🤔

Reply:
What I meant is that at other places in the code we use an int random and see whather it is above a certain percentage. Then you can fine tune, how often it happens.

Increasing the process count is maybe not so advisable. Basically, it is tuned that it should take about a minute on each CI run.

Reply:
Just leaving my two cents regarding CI: Since they now all run in parallel, the engine has a lot of time budget (> 5 additional minutes) until it would slow down the CI.

Reply:
Just leaving my two cents regarding CI: Since they now all run in parallel, the engine has a lot of time budget (> 5 additional minutes) until it would slow down the CI.

Reply:
@pihme Do you have ideas on what a sensible weight for starting anywhere would be? I am thinking 1 in 3 could start at a random place. Since we have the time in the CI I'd like to increase the process count to 6. This way the step would take ~1 minute more, and we'll be testing 4 default starts and 2 random starts.